val foo: Int by throw NullPointerException()